src/auth/useRegisterPasswordlessUser.ts

```diff
@@ -39,6 +40,9 @@ export function useRegisterPasswordlessUser(): UseRegisterPasswordlessUserType {
       if (!setClient) {
         throw new Error("No client context");
       }
+      if (widget) {
+        throw new Error("We never register passwordless user in widget");
```

Comment:
Instead of throwing an error, can we avoid calling this function in widget mode? I think I would find that cleaner.

Reply:
Having unecassary errors in the consol is indeed not the cleanest.
I kept the error however but additionally added a condition where the method is called. This gives us insight in when we accidently call it in a widget at some point.